### ISSUE TITLE
Cannot import files from Figma due to missing checkbox in setting in latest AE

### ISSUE THREAD

**Issue by samueljess** [google/sketch2ae#21]
**Describe the bug**
I am asked to tick a checkbox on the settings, but this checkbox was removed in AE 2019.

**To Reproduce**
1. Import layers from Figma
2. Popup is triggered in AE saying I need to tick the checkbox
3. Settings are loaded with no checkbox available

**Screenshots**
![Screen Shot 2020-10-03 at 5 26 09 PM](https://user-images.githubusercontent.com/3030587/94988220-39f10300-059e-11eb-8d9e-bf18ea074f6a.png)

**Desktop**
 - OS: MacOS 10.15.5
 - AE: 17.1.4

**Comment by adamplouff** [comment]
Good catch. I need to reword that popup. Are layers still building ok, or are images missing?

Adobe moved this pref under the **Scripting and Expressions** tab of the Ae prefs. If you enable it there does that clear things up?

<img width="993" alt="Screen Shot 2020-10-03 at 7 55 02 AM" src="https://user-images.githubusercontent.com/8580225/94990977-f373b800-054d-11eb-8063-2066d2b32dc7.png">

**Comment by samueljess** [comment]
Thanks for the quick reply. It was not working, it got caught in a loop of pressing "OK" on the preferences panel and then showing the error and sending me to preferences. But at the time I had not found where they moved the checkbox to. I checked it now and it's working fine. Thanks for your help.
